README.md

```diff
@@ -27,8 +27,13 @@ tart run ventura-vanilla
 3. Allow SSH. General -> Sharing -> Remote Login & Screen Sharing
 4. Display -> Advanced -> Prevent from sleeping
 5. Open Safari. Preferences -> Advanced -> Show Developer menu. Develop -> Allow Remote Automation.
-6. Sisable SIP. Run `tart run --recovery ventura-vanilla` -> Options -> Utilities menu -> Terminal -> `csrutil disable`.
-7. Run `sudo visudo` in Terminal, find `%admin ALL=(ALL) ALL` add `admin ALL=(ALL) NOPASSWD: ALL` to allow sudo without a password.
+6. (Optional, depends on your needs) Disable SIP. Run `tart run --recovery ventura-vanilla` -> Options -> Utilities menu -> Terminal -> `csrutil disable`.
+7. Enable passwordless `sudo`:
+    1. Run `sudo visudo /private/etc/sudoers.d/admin-passwordless` in Terminal.
+    2. Add `admin ALL = (ALL) NOPASSWD: ALL` to allow `sudo` without a password.
+    3. `:wq` to write the file and quit.
+    4. `sudo visudo -c` to verify your new file parsed OK.
```

Comment:
I wonder if simply calling the `visudo` in the first step should be enough as it "checks for syntax errors before installing the edited file", according to it's manual page.

Reply:
My habit is to drop those files in via `ansible` or otherwise, though, so having the check at the end is muscle-memory for me now. Last place I was at I submitted cross-platform support to `goss` and I think added a `command` assertion.

I've bricked my admin user enough times; I find the file format is not what one might call intuitive.

I'd be happy to change it if you'd prefer; WDYT?

Reply:
For Ansible there's a [`sudoers` module](https://docs.ansible.com/ansible/latest/collections/community/general/sudoers_module.html), which does the check via `visudo` automatically.

If we're considering a manual VM configuration, calling `sudo visudo -c` after `visudo` makes no sense, since the latter already validates the rules before overwriting the file (see `visudo` manual page).

Let's put it another way: `sudo` configuration validation needs to happen **before** that configuration file gets created or overwritten. Otherwise at the point in which you're going to call `sudo visudo -c`, you might be dealing with an already "bricked" sudo.

Do you think it still worth keeping it?

Reply:
Not now I learn ansible has a `sudoers` module; thanks!